### PR TITLE
remove parameters; clean inflation related code in polkadot runtime

### PR DESCRIPTION
**Pull request by kianenigma**
output of the test: 

```
Oct 30 12:37:54.179  INFO runtime::polkadot: total-staked = 848019558.366 DOT (8,480,195,583,663,520,091)    
Oct 30 12:37:54.179  INFO runtime::polkadot: total-issuance = 1512066550.617 DOT (15,120,665,506,175,654,580)    
Oct 30 12:37:54.179  INFO runtime::polkadot: staking-rate = 56.0834811152987811%    
Oct 30 12:37:54.179  INFO runtime::polkadot: era-duration = 86400000    
Oct 30 12:37:54.179  INFO runtime::polkadot: maxStakingRewards = None    
Oct 30 12:37:54.179  INFO runtime::polkadot: 💰 Inflation ==> staking = 279477.810 DOT (2,794,778,104,198,508) / leftover = 49319.613 DOT (493,196,136,035,030)    
Oct 30 12:37:54.179  INFO runtime::polkadot: inflation_rate runtime API: InflationInfo { inflation: 7.9368900607918775%, next_mint: (2794778104198508, 493196136035030) }    
```

### CODE REVIEW

File: relay/polkadot/src/lib.rs [hunk 1]
```diff
@@ -2281,27 +2196,17 @@ impl Runtime {
...
 		let era_duration = 24 * (HOURS as Moment) * MILLISECS_PER_BLOCK;
 		let next_mint = <Self as pallet_staking::Config>::EraPayout::era_payout(
 			staked,
 			stake_able_issuance,
 			era_duration,
 		);
+		// reverse-engineer the current inflation by looking at the total minted against the total
+		// issuance.
+		let inflation =
+			Perquintill::from_rational((next_mint.0 + next_mint.1) * 365, stake_able_issuance);
```
Comment (ggwpez) [inline relay/polkadot/src/lib.rs:2209]: The era_payout code uses 365.25 (*36525 / 100) normally for a year.

File: CHANGELOG.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Chain-spec generator: propagate the `on_chain_release_build` feature to the chain-spec generator. Without this the live/genesis chain-specs contain a wrongly-configured WASM blob ([polkadot-fellows/runtimes#450](https://github.com/polkadot-fellows/runtimes/pull/450)).
 - Adds a migration to the Polkadot Coretime chain to fix an issue from the initial Coretime migration. ([polkadot-fellows/runtimes#458](https://github.com/polkadot-fellows/runtimes/pull/458))
 - Adds migrations to restore currupted staking ledgers in Polkadot and Kusama ([polkadot-fellows/runtimes#447](https://github.com/polkadot-fellows/runtimes/pull/447))
+- Fix `experimental_inflation_info` in Polkadot and remove unused code (https://github.com/polkadot-fellows/runtimes/pull/497)
```
Comment (ggwpez) [inline CHANGELOG.md:29]: 1.3.3 is already out? I think it should go to the `Unchanged` section.

**Review by ggwpez — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (kianenigma) [reply CHANGELOG.md]: 🤦

**Review by bkchr — APPROVED** [review]

**Review by muharem — APPROVED** [review]

**Review by acatangiu — APPROVED** [review]

**Pull request merged**